app/[blog]/page.js

```diff
@@ -1,6 +1,59 @@
-const Blog = () => {
-  // TODO
-  return <div>Blog Page</div>;
+import Image from "next/image";
+
+import kid from "../../public/kid.png";
+import styles from "./page.module.css";
+
+const Blog = (props) => {
```

Comment:
should this be moved to `BlogPost`? here: https://github.com/Hack-Weekly/tangerine-moose-blog/blob/7e4142a160fa866da54f33faca6eb15b85954ea9/app/%5Bblog%5D/%5Bpost%5D/page.js#L122

Reply:
Yeah I just noticed that. So is that the Blog Post? 
What page was I working on?

Reply:
Blog here would be the list of all blog posts as cards, user info/bio at the top. so i think the url would be `/[blogName]` and blog posts are linked here as `/[blogName]/[blogPostSlug]`

Reply:
so i guess the about the author section would stay, and the blog post would go into a card with the text cut off

Reply:
Ok.

I'll just make the blog home page then.
Your blog post has a lot going right now haha